src/odemis/gui/conf/file.py

```diff
@@ -248,6 +248,7 @@ def __init__(self):
         # is completed.
         self.default.set("acquisition", "fn_ptn", u"{datelng}-{timelng}")
         self.default.set("acquisition", "fn_count", "0")
+        self.default.set("acquisition", "overlap", "0.0")
```

Comment:
I don't know how the concept works, so maybe a stupid question. Does this also work for any other type of microscope?

Reply:
yes, you could now use an overlap constant in any microscope, which you can access with `acqui_conf = conf.get_acqui_conf()`

Reply:
Thanks - that is nice! And if you don't want an overlap it also works with any microscope right?

Reply:
yes